src/common/procedure/src/options.rs

```diff
@@ -26,13 +27,16 @@ pub struct ProcedureConfig {
     /// Initial retry delay of procedures, increases exponentially.
     #[serde(with = "humantime_serde")]
     pub retry_delay: Duration,
+    /// `None` stands for no limit.
+    pub max_value_size: Option<ReadableSize>,
```

Comment:
You might need `NoneAsEmptyString`
https://github.com/GreptimeTeam/greptimedb/blob/a61fb98e4a811cd96cde5028bb24273cd95646b3/src/mito2/src/config.rs#L277-L278